samza-core/src/main/java/org/apache/samza/processor/SamzaContainerController.java

```diff
@@ -60,16 +60,14 @@
    * @param taskFactory         Factory that be used create instances of {@link org.apache.samza.task.StreamTask} or
    *                            {@link org.apache.samza.task.AsyncStreamTask}
    * @param containerShutdownMs How long the Samza container should wait for an orderly shutdown of task instances
-   * @param processorId         Id of the processor
    * @param metricsReporterMap  Map of metric reporter name and {@link MetricsReporter} instance
    */
   public SamzaContainerController(
       Object taskFactory,
       long containerShutdownMs,
-      String processorId,
       Map<String, MetricsReporter> metricsReporterMap) {
     this.executorService = Executors.newSingleThreadExecutor(new ThreadFactoryBuilder()
-        .setNameFormat("p" + processorId + "-container-thread-%d").build());
+        .setNameFormat("container-thread-%d").build());
```

Comment:
Mm.. Good point. I think I was simply re-factoring and didn't see the need for processorId except debug logging. But multiple processors within the same JVM will make it hard to differentiate since the threadpool is scoped within each StreamProcessor. 

Let me fix this. Thanks for pointing it out.